docs/manual/mustcall-checker.tex

```diff
@@ -0,0 +1,164 @@
+\htmlhr
+\chapterAndLabel{Must Call Checker}{mustcall-checker}
+
+The MustCall Checker tracks the methods that an object should call before it is deallocated
+and produces a conservative overapproximation of the set of methods that might
+actually need to be called on an object.
+It is intended to be run as a subchecker of another checker: on its own, it does not
+enforce any rules other than subtyping.
+
+For example, an object whose type is \<java.io.OutputStream> might
+have an obligation to call the \<close()> method, to release an underlying file resource. Or,
+such an \<OutputStream> might not have such an obligation, if the underlying resource is
+a byte array. Either of these obligations can be represented by the static type
```

Comment:
This is strictly speaking true, but it is confusing here.
The latter is also `@MustCall({})` and is better to say that fact.
Leave subtyping subtleties until later.